### DISCUSSION TITLE
Why do preview images look worse than when directly inferencing in txt2img?

### DISCUSSION THREAD

**Discussion by Jonseed** [d8ahazard/sd_dreambooth_extension#209] [Q&A]
Often the preview images seem to look much worse than when I am directly inferencing in txt2img. Do I just need to devise a better preview prompt, or is there a reason why preview images are lower quality?

**Comment by Slug-Cat** [comment]
I have also noticed this. I have no idea why, but it kind of defeats the whole point of a preview.

**Comment by sgsdxzy** [comment]
Txt2img in webui contains tons of hack by AUTOMATIC1111, and the preview just uses the default diffuser pipeline.
Once I tried to use webui routines in dreambooth but I find it too difficult to adapt.

**Comment by Art4DD** [comment]
Preview images uses the Scheduler that you select when creating the model as the Sampling method. This is usually a DIMM.
Webui uses "Euler a" by default.

In fact, it's hard to say which Sampling method is better. They just render the images differently.

If you want to see the same thing in txt2img as in Preview images, then you need to select DIMM, set the seed value to 0, and enter the instance promt used in training.
